src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -561,47 +603,28 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
       try {
         performTaskWorkUnsafe();
       } catch (error) {
-        isPerformingTaskWork = false;
         trapError(nextUnitOfWork, error, false);
       }
     }
 
+    // Unschedule the roots that fataled so that we can later schedule
+    // other updates to the same roots and potentially replace them.
```

Comment:
I want to continue working on the other roots in the same frame even if some failed, and only throw errors after all work is done (or frame is over). If I don't unschedule failed roots, `findNextUnitOfWork` will find them again, and we'll get into an infinite loop handling their errors again, then finding them again, and so on.